### ISSUE TITLE
addressbook not activated

### ISSUE THREAD

**Issue by equaeghe** [equaeghe/infcloud#23]
I have a single addressbook; I've added it to `globalActiveAddressbookCollections` and `globalLoadedAddressbookCollections`, but it does not activate on load: I have to click the checkbox to do so on each login. I have `settingsAccount=true`.

**Comment by equaeghe** [comment]
In the Chromium console, on startup I see the error

```
https://baikal.<domain>/calcard.php/addressbooks/<username>/default/ Failed to load resource: the server responded with a status of 403 (Forbidden)
```

This could explain the fact that the default address book is not activated on startup. It is not clear yet why I get this 403.

However, this 403 does not happen on every login, so that may just be an instance of issue #9.

**Comment by equaeghe** [comment]
Not fixed in 0.12.1.

**Comment by equaeghe** [comment]
Fixed in 0.13.0.

**Comment by alryaz** [comment]
Not fixed in 0.13.1
![image](https://cloud.githubusercontent.com/assets/989479/10817461/ffd04526-7e4a-11e5-9c18-5e978ade8915.png)
